distribution-core/pom.xml

```diff
@@ -353,6 +353,14 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- In provided scope, so that covergae is correctly reported but the jar is not packaged for distribution -->
```

Comment:
Small typo here in **coverage**

Maybe we should move this comment to explain how to use scope. For example, the previous dependency is also in this scope.

Reply:
Fixed typo. About moving the comment, actually it refers to this specific dependency